ci/jenkins/jobs/macros.yaml

```diff
@@ -15,6 +15,15 @@
             DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
             ./ci/jenkins/test-vmc.sh --testcase integration --coverage --codecov-token "${CODECOV_TOKEN}" --registry ${DOCKER_REGISTRY}
 
+- builder:
```

Comment:
I didn't find a way to comment unchanged lines in Github. This comment is for line 1.

I think we need to update the builder-job-updater to delete projects-lab.yaml before triggerring jenkins-jobs update -r.

Or if you have time, we can modify the builder-job-updater by adding a Jenkins-job-builder variable jenkins_location. In projects-cloud.yaml, we set `jenkins_location: cloud`, and in projects-lab.yaml, we add an updater job and set `jenkins_location: lab`. Then in this builder shell script, it can delete the projects-lab.yaml or projects-cloud.yaml according to jenkins_location value before updating the jobs.

Reply:
Thanks, updated. Now job updater deletes project files according to jenkins_location.